src/qumin/representations/__init__.py

```diff
@@ -161,7 +166,7 @@ def check_cells(cells, par_cols):
     def parse_cell(cell):
         if not cell:
             return cell
-        forms = [Form(f) for f in cell.split(";")]
+        forms = [Form(form_dic[f], f) for f in cell]
```

Comment:
`Form(form_dic[f], form_id=f)`

Reply:
Yes, it's a style distinction only, but I think it's bad style to be implicit when you can be explicit.